std/typecons.d

```diff
@@ -698,9 +732,45 @@ template Tuple(Specs...)
         }
     }
 }
+///
```

Comment:
Nitpick: please insert blank line before the ddoc marker for each unittest block.